src/main/java/dk/sdu/seb05/semesterprojekt/PresentationLayer/CreatePageController.java

```diff
@@ -187,48 +161,21 @@ else if (!errorHandler()) {
     }
 
     public String monthToString(int month){
-        String returnMonth;
-        switch (month){
-            case 1:
-                returnMonth = "Januar";
-                break;
-            case 2:
-                returnMonth = "Februar";
-                break;
-            case 3:
-                returnMonth = "Marts";
-                break;
-            case 4:
-                returnMonth = "April";
-                break;
-            case 5:
-                returnMonth = "Maj";
-                break;
-            case 6:
-                returnMonth = "Juni";
-                break;
-            case 7:
-                returnMonth = "Juli";
-                break;
-            case 8:
-                returnMonth = "August";
-                break;
-            case 9:
-                returnMonth = "September";
-                break;
-            case 10:
-                returnMonth = "Oktober";
-                break;
-            case 11:
-                returnMonth = "November";
-                break;
-            case 12:
-                returnMonth = "December";
-                break;
-            default:
-                throw new IllegalStateException("Unexpected value: " + month);
-        }
-        return returnMonth;
+        return switch (month) {
+            case 1 -> "Januar";
+            case 2 -> "Februar";
+            case 3 -> "Marts";
+            case 4 -> "April";
+            case 5 -> "Maj";
+            case 6 -> "Juni";
+            case 7 -> "Juli";
+            case 8 -> "August";
+            case 9 -> "September";
+            case 10 -> "Oktober";
+            case 11 -> "November";
+            case 12 -> "December";
+            default -> throw new IllegalStateException("Unexpected value: " + month);
```

Comment:
Virkelig petitesse, men det burde nok være en `IllegalArgumentException`.
IllegalStateException betyder at programmet er i en ugyldig state, eller ugyldig enviroment. Men det er jo bare et ugyldigt argument, og dér passer `IllegalArgumentException`.

Reply:
That shall be fixed posthaste